Algorithm.CSharp/LiquidETFUniverseFrameworkAlgorithm.cs

```diff
@@ -0,0 +1,107 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using QuantConnect.Algorithm.Framework.Alphas;
+using QuantConnect.Algorithm.Framework.Execution;
+using QuantConnect.Algorithm.Framework.Portfolio;
+using QuantConnect.Algorithm.Framework.Selection;
+using QuantConnect.Brokerages;
+using QuantConnect.Data;
+using QuantConnect.Data.UniverseSelection;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace QuantConnect.Algorithm.CSharp
+{
+    /// <summary>
+    /// Basic template framework algorithm uses framework components to define the algorithm.
+    /// Liquid ETF Competition template
+    /// </summary>
+    /// <meta name="tag" content="competition" />
+    /// <meta name="tag" content="alpha stream" />
+    /// <meta name="tag" content="using quantconnect" />
+    public class LiquidETFUniverseFrameworkAlgorithm : QCAlgorithm
+    {
+        // List of symbols we want to trade. Set it in OnSecuritiesChanged
+        private readonly List<Symbol> _symbols = new List<Symbol>();
+
+        public override void Initialize()
+        {
+            // Set Start Date so that backtest has 5+ years of data
+            SetStartDate(2014, 11, 1);
+            // No need to set End Date as the final submission will be tested
+            // up until the review date
+
+            // Set $1m Strategy Cash to trade significant AUM
+            SetCash(1000000);
+
+            // Add a relevant benchmark, with the default being SPY
+            AddEquity("SPY");
```

Comment:
`SPY` is already the default benchmark, there is no need to manually add it, Lean will add it. Is there another reason we are adding `SPY`?

Reply:
In the examples for the competition, we have been adding `SPY` because it is used in Scheduled Events. I kept the `Initialize` as close as possible of competition templates.

Reply:
I guess we could stop using it so that SPY isn't added as a tradeable security